src/components/book.jsx

```diff
@@ -1,18 +1,31 @@
-import React, { Component } from 'react';
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { bookRemoved } from '../redux/books/books';
+import Categories from './Categories';
 
-class Book extends Component {
-
- render() { 
-  const { title , author } = this.props;
+function Book({ book }) {
+  const dispatch = useDispatch();
   return (
-    <>
     <section className="container">
-     <h2>{title}</h2>
-     <p>{author}</p>
+      <h2>{book.title}</h2>
+      <p>{book.author}</p>
+      <Categories />
```

Comment:
- [ ] Kindly remove this Categories component from here. It is displayed and repeated on the Books homepage.

  ![ifdo](https://user-images.githubusercontent.com/81468675/207848359-8b3f223b-309e-4973-9477-b5c6281644bc.jpg)

- [ ] When I click the "categories" on the navbar, nothing happens. I should be navigated to the categories component. In your **App.js** component kindly define your Categories route accordingly and ensure that when the "check status" button is clicked the text is displayed.

Reply:
thanks for your time and your effort, and thanks for pointing out the fact in details. I will improve my code with your suggestions.